api/app/controllers/review.controller.ts

```diff
@@ -114,67 +114,28 @@ export async function findAll(req: Request, res: Response) {
     return;
   }
 
-  let groupedCondition = {};
-  const currentTime = new Date().getTime();
-
-  const userIdCondition = { userId: { [Op.eq]: user.id } };
-
-  // add condition if query exists
-  if (Boolean(user.id)) {
-    groupedCondition = { ...groupedCondition, ...userIdCondition };
-  }
-  if (queryStatus) {
-    switch (queryStatus) {
-      case QUERY_REVIEW_STATUS.ACTIVE:
-        const activeCondition = buildIsActiveCondition(currentTime);
-        groupedCondition = { ...groupedCondition, ...activeCondition };
-        break;
-      case QUERY_REVIEW_STATUS.PENDING:
-        const pendingCondition = buildIsPendingCondition();
-        groupedCondition = { ...groupedCondition, ...pendingCondition };
-        break;
-      case QUERY_REVIEW_STATUS.INACTIVE:
-        const inActiveCondition = buildIsInActiveCondition(currentTime);
-        groupedCondition = { ...groupedCondition, ...inActiveCondition };
-        break;
-      default:
-        break;
-    }
-  }
+  const { condition } = buildCondition(queryStatus, undefined, user.id, undefined, undefined);
```

Comment:
if using object args like I suggested here, it would simplify this function call so we won't need to pass in undefined values here.
```suggestion
  const { condition } = buildCondition({status: queryStatus, userId: user.id});
```